Userland/Libraries/LibWebView/ChromeProcess.cpp

```diff
@@ -27,10 +27,6 @@ class UIProcessClient final
 
 ErrorOr<ChromeProcess> ChromeProcess::create()
```

Comment:
Seems like returning `ErrorOr` is pointless now :)

Reply:
tru!